NEWS_DEV.txt

```diff
@@ -30,6 +30,7 @@ Highlights:
  - Theano now internally uses ``sha256`` instead of ``md5`` to work on systems that forbide ``md5`` for security reason
  - Removed old GPU backend ``theano.sandbox.cuda``. New backend ``theano.gpuarray`` is now the official GPU backend
  - Support more debuggers for ``PdbBreakpoint``
+ - Make sure MKL uses GNU OpenMP
```

Comment:
I would add as a subpoint more details:

matrix dot product (gemm) with numpy 1.13 and mkl from conda could is some cases return wrong results. We have reported the problem upstream and have a work around to have an error raised with information how to fix it to the user.